samples/SampleSender/Program.cs

```diff
@@ -15,7 +15,12 @@ public class Program
 
         public static void Main(string[] args)
         {
-            SendMessagesToEventHub(100).Wait();
+            MainAsync(args).GetAwaiter().GetResult();
+        }
+
+        private static async Task MainAsync(string[] args)
+        {
+            await SendMessagesToEventHub(100).ConfigureAwait(false);
```

Comment:
@jtaubensee @serkantkaraca removing `.ConfigureAwait(false)`. Please consider not about it to the documentation on samples.